src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/MarshalTests.cs

```diff
@@ -3,11 +3,129 @@
 // See the LICENSE file in the project root for more information.
 
 using Xunit;
+using System.Collections.Generic;
+using System.Security;
 
 namespace System.Runtime.InteropServices
 {
     public static class MarshalTests
     {
+        public static IEnumerable<object[]> StringData =>
```

Comment:
Nit: It doesn't really matter, but this could be simplified as:

```c#
public static readonly object[][] StringData =
{
    new object[] { "pizza" },
    new object[] { "pepperoni" },
    new object[] { "password" },
    new object[] { "P4ssw0rdAa1!" },
};
```